packages/realm-server/tests/realm-server-test.ts

```diff
@@ -2200,13 +2200,13 @@ module('Realm server serving multiple realms', function (hooks) {
     async () => await loader.import(`${baseRealm.url}card-api`),
   );
 
-  hooks.beforeEach(async function () {
+  hooks.before(async function () {
```

Comment:
The full indexing test mutates the state of the realms. and these realms are shared between multiple tests--if we don't recreate the realm server inbetween each test, how can we be sure realm state from the full indexing test doesn't bleed into the other test in this module?

Reply:
I didn't think of that before. Will switch it back to `beforeEach`

Reply:
Fixed in https://github.com/cardstack/boxel/pull/1306/commits/48206862054e1f4ee7b7b04d6af11b5ec3e4819d